src/components/templates/ConversationSpace/ConversationSpace.scss

```diff
@@ -252,6 +250,7 @@
     display: grid;
     grid-template-columns: repeat(12, 1fr);
     grid-gap: 1rem;
+    margin: 10px 70px;
```

Comment:
What are the changes here based on?

Reply:
also jazzbar

Reply:
Makes sense. If it looks fine then feel free to mark this resolved.